### PR TITLE
Add firehose logs enable[CDS-369]

### PR DESCRIPTION
**Pull request by Avivsc-corl**
Added firehose logs to the firehose metrics.

### CODE REVIEW

File: examples/firehose/firehose-enable-logs/README.md
Comment (royfur) [inline examples/firehose/firehose-enable-logs/README.md:1]: if this example is for logs only why we dont disable metric?

File: modules/firehose/README.md
Comment (royfur) [inline modules/firehose/README.md:1]: need to add Authors

File: modules/firehose/variables.tf
Comment (royfur) [inline modules/firehose/variables.tf:1]: Need to add option for custom url

File: examples/firehose/firehose-enable-logs/README.md
Comment (Avivsc-corl) [reply examples/firehose/firehose-enable-logs/README.md:1]: Updated - added metric_enable = false

File: modules/firehose/main.tf
Comment (ryantanjunming) [inline modules/firehose/main.tf:1]: remove index_metric please, unused variable

**Review by ryantanjunming — APPROVED** [review]

File: modules/firehose/main.tf
Comment (Avivsc-corl) [reply modules/firehose/main.tf:1]: Done

**Pull request merged**